### ISSUE TITLE
Feature: Visiting repos not listed in the goals issues should present a tracking button

### ISSUE THREAD

**Issue by bdougie** [open-sauced/open-sauced#790]
### Is your feature request related to a problem? Please describe.

This is a feature. 

Adding the ability to view repositories that are not listed in the tracking(goals) repository. 

### Describe the solution you'd like

When visiting an untracked project, I would like to be asked to track it and not show the note form or other sidebar items.

### Describe alternatives you've considered

Hide the following if the project is not the user tracking (goals) repo:
- notes box
- contributions
- fork button

Show the following:
- tracking repo as a goal button instead of fork button.

### Additional context

This is dependent on #789 this being fixed

**Comment by bdougie** [comment]
This is a nice to have but would introduce more bugs than needed because of how the router works.
